docs/en/developers/01_First_steps.md

```diff
@@ -6,6 +6,10 @@
 
 **TODO**
 
+# Extensions
+
+If you want to create your own FreshRSS extension, either use the [incomplete english documentation](03_Backend/05_Extensions.md) or try the more complete [french version](../../fr/developers/03_Backend/05_Extensions.md).
```

Comment:
Unfortunately [DeepL](https://www.deepl.com/translator) (the best for French, at least *to* French) doesn't offer full page translation, but it could be an idea to include a link to some machine translation? But if you're not touching the extensions file I'll just start a separate PR to translate the remainder of the French and you can leave out the remark about completeness.

Reply:
DeepL is the best for german as well, but never tried it with technical docus. Actually I started the english version before DeepL came out, but using Google Translate was a nightmare ... I could only concentrate on the possible meaning of Googles translations instead of the actual content.